jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionItemView.java

```diff
@@ -0,0 +1,163 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.cm.client.actions;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+
+import com.google.gwt.user.client.TakesValue;
+import org.jboss.errai.common.client.api.IsElement;
+import org.jboss.errai.common.client.dom.Div;
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.common.client.dom.Span;
+import org.jboss.errai.common.client.dom.UnorderedList;
+import org.jboss.errai.databinding.client.api.DataBinder;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.AutoBound;
+import org.jboss.errai.ui.shared.api.annotations.Bound;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.cm.client.events.CaseStartAdhocFragmentEvent;
+import org.jbpm.workbench.cm.client.util.AbstractView;
+import org.jbpm.workbench.cm.util.CaseActionStatus;
+import org.jbpm.workbench.cm.model.CaseActionSummary;
+
+
+import static org.jboss.errai.common.client.dom.DOMUtil.*;
+import static org.jboss.errai.common.client.dom.Window.*;
+import static org.jbpm.workbench.cm.client.resources.i18n.Constants.*;
+
+
+@Dependent
+@Templated
+public class CaseActionItemView extends AbstractView<CaseActionListsPresenter> implements TakesValue<CaseActionSummary>, IsElement {
+
+    @Inject
+    protected TranslationService translationService;
+    @Inject
+    @DataField("action-bullet")
+    Span bullet;
+    @Inject
+    @DataField("action-name")
+    @Bound
+    Span name;
+    @Inject
+    @DataField("action-status")
+    @Bound
+    Span status;
+    @Inject
+    @DataField("list-group-item")
+    Div listGroupItem;
+    @Inject
+    @DataField("actions-dropdown")
+    Div actions;
+    @Inject
+    @DataField("actions-items")
+    UnorderedList actionsItems;
+    @Inject
+    @AutoBound
+    private DataBinder<CaseActionSummary> caseActionSummary;
+    private Event<CaseStartAdhocFragmentEvent> startAdhocFragmentEvent;
+
+    @Override
+    public void init(final CaseActionListsPresenter presenter) {
+        this.presenter = presenter;
+    }
+
+    @Override
+    public HTMLElement getElement() {
+        return listGroupItem;
+    }
+
+    @Inject
+    public void setCommentEditEvent(final Event<CaseStartAdhocFragmentEvent> startAdhocFragmentEvent) {
```

Comment:
Is this method name a typo?
Shouldn't name be something along the lines of "setStartAdhocFragmentEvent"?